<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>NN1050</family>
    <pointsize>26</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tab_bar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="jog_tab">
       <attribute name="title">
        <string>JOG</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </widget>
      <widget class="QWidget" name="joystick_tab">
       <attribute name="title">
        <string>STICK</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="ab_tab">
       <attribute name="title">
        <string>A-B</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="sequence_tab">
       <attribute name="title">
        <string>SEQ</string>
       </attribute>
       <widget class="QPushButton" name="soft_key_A">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>420</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>A</string>
        </property>
       </widget>
       <widget class="QPushButton" name="soft_key_C">
        <property name="geometry">
         <rect>
          <x>403</x>
          <y>420</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>SELECT</string>
        </property>
       </widget>
       <widget class="QPushButton" name="soft_key_B">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>420</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>B</string>
        </property>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>169</x>
          <y>200</y>
          <width>271</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="parameters">
         <item row="1" column="1">
          <widget class="QLabel" name="speed_value">
           <property name="text">
            <string>0.1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="speed_label">
           <property name="text">
            <string>SPEED</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="speed_unit">
           <property name="text">
            <string>m/s</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="deflection_label">
           <property name="text">
            <string>DEFLECTION</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="deflection_value">
           <property name="text">
            <string>0.1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="deflection_unit">
           <property name="text">
            <string>m</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="settings_tab">
       <attribute name="title">
        <string>SYS</string>
       </attribute>
       <widget class="QPushButton" name="initialize_button">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>321</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Initialize Steppers</string>
        </property>
       </widget>
       <widget class="QPushButton" name="homing_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>321</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Run Homing Sequence</string>
        </property>
       </widget>
       <widget class="QPushButton" name="joystick_calibration_button">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>190</y>
          <width>161</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Joystick
Calibration</string>
        </property>
       </widget>
       <widget class="QPushButton" name="estop_button">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>270</y>
          <width>161</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>NN1050</family>
          <pointsize>26</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>EMERGENCY
STOP</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="quick_test_button">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>110</y>
          <width>161</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Quick Test</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sys_exit_button">
        <property name="geometry">
         <rect>
          <x>399</x>
          <y>350</y>
          <width>161</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>sys.exit()</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
